tachyon/zk/plonk/permutation/permutation_prover_impl.h

```diff
@@ -119,6 +119,9 @@ template <typename PCS>
 void PermutationProver<Poly, Evals>::Evaluate(
     ProverBase<PCS>* prover,
     const PermutationOpeningPointSet<F>& point_set) const {
+  // NOTE(chokobole): The ORDER IS IMPORTANT!!!
```

Comment:
```suggestion
  // The ORDER IS IMPORTANT!!!
```
I thought we were removing NOTE(author) comments. https://github.com/kroma-network/tachyon/commit/98db4aa004b6202d9ca0660b74456f30f1dc2102